parsl/executors/high_throughput/executor.py

```diff
@@ -714,8 +714,15 @@ def scale_in(self, blocks=None, block_ids=[], force=True, max_idletime=None):
 
         return self._filter_scale_in_ids(to_kill, r)
 
-    def _get_job_ids(self) -> List[object]:
-        return list(self.blocks.values())
+    def _get_block_and_job_ids(self) -> Tuple[List[str], List[Any]]:
+        # Not using self.blocks.keys() and self.blocks.values()
```

Comment:
what does this comment mean? the code *does* use self.block.keys()

Reply:
Wanted to say it does not use `self.blocks.keys()` and `self.blocks.values()` simultaneously.
E.g. `self.block` may be changing when invoking `self.blocks.keys()` and `self.blocks.values()` in between, and thus they returns different lengths.